### DISCUSSION TITLE
Give /back an icon in /homes

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#9268] [Enhancement]
We have an icon for /dback, but there is no icon for /back that shows, so I have to give players both dback and back just to get an icon in /homes
The fact they are seperate would be great to have a seperate icon for back saved please, then I can fully remove dback from players.

**Comment by Zrips** [comment]
Will add separate button with next update

**Reply by TomLewis** [reply]
Legand!
